src/main/java/edu/berkeley/ground/util/ElasticSearch.java

```diff
@@ -0,0 +1,103 @@
+package edu.berkeley.ground.util;
+
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.berkeley.ground.exceptions.GroundException;
+import edu.berkeley.ground.model.models.Tag;
+import org.elasticsearch.action.index.IndexResponse;
+import org.elasticsearch.client.Client;
+import org.elasticsearch.node.Node;
+import static org.elasticsearch.node.NodeBuilder.nodeBuilder;
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.search.SearchHit;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ElasticSearch {
+  private static Node node;
+  private static Client client;
+  private static String clusterName = "groundv1";
+
+
+
+  public static boolean connectElasticSearch() throws GroundException {
+    try {
+      node = nodeBuilder().clusterName(clusterName).node();
+      client = node.client();
+    } catch (Exception e) {
+      throw new GroundException("ElasticSearch encountered an error while trying to connect");
+    }
+    return !node.isClosed();
+
+  }
+
+  public static boolean insertElasticSearch(Tag tag, String table) throws GroundException {
+
+
+    ObjectMapper mapper = new ObjectMapper();
+
+    try {
+      String json = mapper.writeValueAsString(tag);
+      IndexResponse response = client.prepareIndex(clusterName, table, Long.toString(tag.getId()))
+        .setSource(json).get();
+      client.admin().indices().prepareRefresh().execute().actionGet(); // need to refresh index with new inserted item
+
+      return response.isCreated();
+
+    } catch (JsonProcessingException e) {
+      e.printStackTrace();
```

Comment:
Printing the stack trace isn't the right thing to do. We should pass the error information through with the exception that we throw.